mypy/semanal_pass1.py

```diff
@@ -154,6 +154,17 @@ def visit_func_def(self, func: FuncDef) -> None:
         func.is_conditional = sem.block_depth[-1] > 0
         func._fullname = sem.qualified_name(func.name())
         at_module = sem.is_module_scope()
+        if (at_module and func.name() == '__getattr__' and
+                self.sem.cur_mod_node.is_package_init_file() and self.sem.cur_mod_node.is_stub):
+            if isinstance(func.type, CallableType):
+                ret = func.type.ret_type
+                if isinstance(ret, UnboundType) and not ret.args:
+                    sym = self.sem.lookup_qualified(ret.name, func, suppress_errors=True)
+                    # We only interpred package as partial if __getattr__ return type
```

Comment:
`We only interpred` -> `We only interpret a`

Reply:
Also, `if` -> `if the`